### DISCUSSION TITLE
À quoi sert ce produit logiciel ?

### DISCUSSION THREAD

**Discussion by benel** [Hypertopic/HyperGlosae#1] [General]
Durant ce semestre, même si vous serez dans quatre équipes différentes (avec des "clients" – ou plutôt des "parties prenantes"  – différentes), vous allez tous contribuer au même "**produit logiciel**" : _Hyperglosae_.

Autrement dit, **les fonctionnalités que vous développerez pour votre partie prenante seront disponibles également pour les parties prenantes des autres projets**.

Pour réussir à développer ce logiciel ensemble, il est indispensable que chacun d'entre nous partage la même "vision du produit" (_product vision_). 

Au bas de ce message, avec vos propres mots, écrivez ce qui selon vous serait la vision du produit _Hyperglosae_.

**Comment by vallhallalm** [comment]
Selon moi, une vision produit formulée de la manière proposée par le **"vision product statement"** pour le produit _Hyperglosae_ pourrait être :

**Pour** les personnes souhaitant préserver les façons ancestral d'écrire un document textuel quel qu'il soit
**Qui** souhaite rédiger ou éditer et lire des documents présentant des informations liées entre elles grâce à des liens bidirectionnels
Hyperglosae **est un** éditeur de document 
**Qui** intègre le concept de lien bidirectionnel dans l'édition de ces fichiers.
**Au contraire** d'un éditeur de document tel que Microsoft Word,
**Notre produit** propose l'implémentation de l'idée originel de Theodor Holm Nelson adaptée de la façon dont les ouvrages sont rédigés historiquement, et qui n'a pas été intégré dans le concept du "world wide web" actuel et donc par extension dans les logiciels nous faisant concurrence.


Une autre formulation de la vision du produit exprimée sous la forme d'un **slogan** cette fois-ci pourrait être  : 

Rétablir les racines documentaires historiques au sein du monde numérique.

**Comment by lililiane** [comment]
- La vision produit d'Hyperglosae, sous forme de **slogan** serait :
Connaître les références de tous documents numériques

- La vision produit d'Hyperglosae, en utilisant le **vision product statemen**t serait :
HyperGlosae est une infrastructure numérique permettant aux utilisateurs qui souhaitent préserver les méthodes traditionnels de rédaction d'utiliser des liens hypertext. 
HyperGlorae permet de créer un document et d'en générer la bibliographie indépendamment de l'utilisateur, contrairement à Wikipedia.

**Comment by SebBac** [comment]
Résumé en une phrase, je dirais que l'objectif principal du produit Hyperglosae serait de :
<< Permettre de faire apparaître clairement et visuellement les liens entre différents documents webs afin d'en améliorer la compréhension et l'accès aux sources >>.

Le but de Hyperglosae est donc de proposer des façons plus efficaces de sourcer des documents webs afin de grandement améliorer la compréhension de leur contenu en voyant notamment instantanément d'où proviennent les informations de chaque partie et quels parallèles ont pu être faits.

Par exemple, plutôt que de simplement proposer une bibliographie à la fin avec les liens de tous les articles et documents utilisés, Hyperglosae proposerait plutôt directement toutes les sources en parallèle de la lecture _(avec des annotations ou via des liens ouvrant une nouvelle fenêtre reliée au paragraphe concerné par un trait sur l'écran par exemple)_. Ainsi, s'il est difficile de savoir d'où proviennent les informations de la partie **II. b) Plantation des carottes** en parcourant une simple bibliographie, avec Hyperglosae on trouverait instantanément que toutes ces informations sont basées sur le _Traité de l'alliance des petits pois, 1754_ à sa page 42.

Hyperglosae permettra donc d'accéder aux sources plus facilement en les reliant directement aux contenus concernés.

**Comment by RomainUTT** [comment]
**According to the Product Vision Statement, my product vision is:** 
For people who want to create rich documents with reference links.
HyperGlosae is a text editor platform that allows you to add hypertext to any document. 
Unlike other website such as Wikipedia, our product allows you to easily navigate between referenced documents and add references without author validation.

**The slogan would be:**
Enhance documents with its reference links.

**Comment by CorentinPARZ** [comment]
**Slogan**: Facilitate access to references.

**Product statement**: 
For anyone, who needs to efficiently visualize the relationships between documents.
[HyperGlosae](https://github.com/Hypertopic/HyperGlosae#readme) is a bidirectional and parallel documentation web application that allows you to access document references during the reading.
Unlike Wikipedia, you can read all referenced documents that are linked to the original from the original document page, without the author's intervention.
Our product is bidirectional, the two documents are linked together. You can access the referenced document to the original and from the original to the referenced document.

**Comment by benel** [comment]
Merci à @vallhallalm, @lililiane, @SebBac, @RomainUTT et @CorentinPARZ pour leurs réponses.

Pour ceux qui n'ont pas encore répondu : 

1. Pensez bien à mentionner comme @vallhallalm, @lililiane  et @RomainUTT,  la **rédaction** de documents. L'écriture est aussi importante ici que la lecture ;

2. Pour les produits concurrents, les commentaires jusqu'à maintenant mentionnent  _Word_ et _Wikipedia_... Si *Hyperglosae* permet la lecture de documents, c'est probablement que les documents (au moins une partie) seront publics sur le Web, quelles sont alors les catégories de logiciels qui sont utilisées couramment pour cela ?

**Comment by Ziwei-GONG-UTT** [comment]
Pour moi, le point central d'Hyperglosae est le lien bidirectionnel, qui permet à l'utilisateur de trouver l'origine. 

C'est un peu comme poster son avis sous un sujet ou un hashtag, mais il fonctionne sur le principe inverse et de manière opposée. D'après ce que j'ai compris, les utilisateurs postent simplement leurs opinions pertinentes et Hyperglosae les concentrera sur l'article d'origine. De cette manière, il est possible d'éviter de voir des articles en double à différents sites et de protéger les droits des auteurs originaux. Les opinions et les discussions des gens seront également plus ciblées et mieux documentées.

Le style de lecture parallèle me fait penser aux compilateurs markdown, les logiciels comme Typora ou Macdown fonctionnent très bien pour moi.

**Comment by kemplail** [comment]
En se basant sur le **product vision statement**, ma vision du produit serait la suivante :

Pour l’ensemble des utilisateurs qui auraient un besoin de consulter et rédiger des documents liés à d’autres documents, HyperGlosae est une infrastructure hypertexte qui va permettre de rédiger et consulter des documents en simplifiant l’accès, la visualisation et l’ajout de références à d’autres documents, ce qui va faciliter la lecture, la compréhension et la rédaction de ces documents, en adoptant une approche davantage naturelle et adaptée à l’humain par rapport à ce qui existe aujourd’hui notamment sur le web, en reprenant les traditions littéraires existantes. Contrairement à des logiciels permettant de visualiser des documents accessibles en ligne, au format PDF par exemple, comme Adobe Acrobat Reader, le produit permettra de disposer de fonctionnalités mettant en avant des principes hypertextes tels que la notion de bi-directionnalité entre les documents, avec les reverse links par exemple (pour accéder aux documents référençant un document) ou encore la notion de parallélisme (pour afficher côte à côte des documents liés en affichant leurs liens).

Si je devais définir un **slogan**, ce serait peut être : Faciliter la manipulation de documents numériques liés selon une approche humaniste.

**Comment by AVCharpentier** [comment]
## Slogan 
Rétablir tous les liens entre les documents numériques

## Product Vision Statement
**Pour** les lecteurs et les rédacteurs de documents qui sont liés à d'autres documents
**Qui** souhaitent consulter, éditer et ajouter des informations en lien avec celui-ci et de connaitre ces sources.
**Hyperglosae** est une infrastructure numérique.
**Qui** permet aux utilisateurs de créer et de visualiser des liens entre des idées, des concepts et des données, des annotations à travers des documents distincts et des sources variées, offrant ainsi une navigation bidirectionnelle entre ces documents sans l'accord systématique de son auteur pour être établi.
**Contrairement** à des navigateurs, des suites bureautique en ligne, ou plus généralement des lecteurs de documents en ligne qui n'évoquent pas tout le temps leurs sources
**Notre produit** permet de naviguer en plus vers les documents qui utilisent le document en cours de lecture pour être rédigés et de consulter des annotions de ce document.

**Comment by StachoB** [comment]
Pour la **vision du produit** :
Pour les lecteurs de documents numérique qui souhaitent rétablir les liens entre documents, Hyperglosae est une plateforme permettant l'accès et la consultation simplifiée de références citées au sein d'un documents durant sa lecture. Contrairement à des logiciels permettant la consultation de documents en ligne classiques, ce produit proposera un référencement des sources plus complet en y intégrant des fonctionnalités plus poussées telles que les reverses links ou encore une navigation et lecture optimale entre documents référencés.

Je peux proposer le **slogan** : Faciliter la consultation de références entre documents numériques

**Comment by MCtzr** [comment]
Selon moi, un **slogan** de la vision du produit Hyperglosae serait : permettre et faciliter le référencement des documents numeriques.

En suivant le "Product Vision Statement", 
**Pour** les auteurs et lecteurs de documents en ligne
**Qui** ont nécessité d'ajouter des référencement et des sources dans leurs documents en ligne.
Hyperglosae est une infrastructure numérique participative qui est mise en place pour les lecteurs et auteurs de documents en ligne.

**Contrairement** aux editeurs de texte habituels et documents en ligne ne référençant pas toujours leurs sources,
**Notre outil** est met en avant le concept de liens hypertexts bidirectionnels créés à la fois par les auteurs et les lecteurs permettant une meilleure interconnexion des documents entre eux et ainsi une meilleure expérience de lecture.

**Comment by julien-millour** [comment]
Voici selon moi la vision du produit *Hyperglosae*:

Pour les rédacteurs et lecteurs de documents en ligne,
Qui veulent lire ou ajouter des informations supplémentaires aux documents numériques,
*Hyperglosae* est une infrastructure hypertexte 
Qui permet d'ajouter du contenu hypertexte à des documents
Contraitement à Google Docs,
Notre produit propose plusieurs types de contenu hypertexte et pas seulement un simple commentaire.

**Comment by Rochijr** [comment]
Selon moi, une vision du produit _Hyperglosae_ pourrait être :
Rendre les références d'un document claires et faciles d'accès.

Ainsi, dans cette idée, le "Product Vision Statement" auquel je pense pour ce produit serait :
**Pour** toutes personnes accédant à un document,
**Qui** désirent ajouter ou modifier des références / sources à un document,
_Hyperglosae_ **est une** plateforme numérique,
**Permettant** la consultation et/ou l'ajout simplifié des références citées dans un document lors de sa lecture ou de sa rédaction.
**Contrairement** aux éditeurs de documents comme Microsoft Word ou des lecteurs de documents en ligne,
**Notre produit** permet de naviguer aisément entre les documents référencés et d'en ajouter de nouveaux simplement, sans la validation obligatoire de l'auteur.

**Comment by GagatheRollex** [comment]
Selon moi, un **slogan** qui pourrais correspondre à la vision produit d'_Hyperglosae_ serait : référencer et structurer les sources du numérique.
Le "Product Vision Statement" serait : 

- **Pour** tous les auteurs et lecteurs de documentation numérique
- **Qui** souhaite modifier ou ajouter des informations aux documents
- **Le produit** _Hyperglosae_ est une outil collaboratif de référencement
- **Qui** permet de consulter et créer des liens hypertextes entre des documents de manière logique et de cette manière créer des annotations, des modifications, des sources ou encore des traductions d'un document source
- **Contrairement** aux éditeurs de texte traditionnels qui ne permettent pas de visualiser facilement des liens entre des documents
- **Le produit** _Hyperglosae_  propose donc une architecture entre des documents numériques grâce à des liens hypertextes bidirectionnels de manière facilité

**Comment by MajedHL** [comment]
selon moi, le **product vision statement** sera le suivant:
**Pour** tous lecteurs, auteurs ou utilisateurs en général
**Qui** souhaitent enrichir leurs documents, en référencant d'autres sources qui permet d'ajouter de la valeur à un document et le personnalisé, d'une manière simple et efficace.
**Le produit** _Hyperglosae_ un outil de documentation.
**Qui** permet de faire référence à d'autres sources, personnaliser les documents, engendrer une bibliographie avec des liens bidirectionnels.
**Contrairement** à tous aux éditeurs de documents traditionnels tels que Microsoft Word, google doc, etc.
**Notre produit** permet aux utilisateurs, auteurs ou lecteurs, de créer une interconnexion organiser et bidirectionnel entre les documents, ce qui permet une meilleur compréhension, lisibilité et expérience en général au lecteur, mais aussi de référencer des sources qui ont servi à la rédaction du document ou peuvent aider à la compréhension du document.

**Comment by marina-pera** [comment]
Une hypothèse de la vision du produit hyperglosae serait "une infrastructure qui permettrait d'exposer clairement les hypertexts"

**Groupe cible:**
- les utilisateurs du web qui consultent un document quelconque sur internet
**Besoin:**
- Le problème à résoudre serait l'écart entre les différentes infrastructures du web pour les hypertexts.
- Cela permettrait de s'y retrouver plus facilement dans les différentes pages web.
**Produit:**
- Le type de produit serait une infrastructure commune pour les documents.
**Valeur:**
- Permettre une même entente entre les différentes manières de présenter les hypertexts.

**Comment by raphaelweis** [comment]
HyperGlosae est un logiciel visant à proposer une implémentation de [la vision de Theodor Nelson concernant les liens hypertextes dans les documents](https://www.youtube.com/watch?v=hMKy52Intac). Voici un exemple de **product vision statement**:
- HyperGloase est développé **Pour** les scientifiques, étudiants, chercheurs, professeurs et tous consommateurs de littérature, de produits culturels, d'arts et de documents en générals.
- HyperGloase **est** un logiciel permettant d'éditer et de consulter des documents de natures diverses.
- HyperGloase est un logiciel **qui** permet à son utilisateur d'utiliser le concept de liens visibles bidirectionnels tels que Theodor Nelson les avait envisagés.
- **Contrairement** aux alternatives disponibles telles que Microsoft Word, HyperGloase permet la consultation et l'édition directe de liens hypertextes bidirectionnels dans les documents, et ce sans accord systématique des auteurs concernés.
- HyperGlosae **permet** de retrouver des virtuellement les façons ancestrales d'éditer et de consulter des documents physiques.

**Comment by ToGoSleep** [comment]
Hyperglosae est un logiciel se basant sur les liens hypertextes qui sont des liens bidirectionnels permettant de consulter et rédiger des documents numériques liés à d'autres documents très facilement par n'importe quelle lecteur.
Contrairement à Microsoft Word, Hyperglosae  permet de naviguer facilement entre ces documents liés et d'ajouter de futurs documents liés facilement grâce aux liens, pouvant permettre aux lecteurs de lire des documents côte à côte, de citer d'autres documents comme les sources ou d'expliquer le document avec d'autres documents.
Hyperglosae permet donc une meilleure comprehension des documents et de les enrichir grâce aux référencements via les liens bidirectionnels entre documents, pour une redaction et une lecture simplifiées de documents.

**Comment by AliceGuib** [comment]
Un **slogan** pour _Hyperglosae_ pourrait être : Enrichir les différentes sources de documentation en les connectant entre elles. 

Un **product vision statement** pour _Hyperglosae_ pourrait être : 

- **Pour :** Les consommateurs ou auteurs de documents culturels (littérature, oeuvres d’art, etc)
- **Besoin :** Faire des liens entre différents documents pour pouvoir les comparer 
- **Quoi :** Le produit Hyperglosae est un outil collaboratif
- **Qui :** Permet à un utilisateur, lorsqu’il consulte un document, de pouvoir consulter le/les documents auxquels il fait référence, ou à l’inverse, consulter les documents qui font référence à celui qu’il consultait initialement. Hyperglosae donne la possibilité aux utilisateurs de créer ces liens librement, sans l’accord de l’auteur ou de l’organisation ôte du document original. Les différents documents liés peuvent être mis côte à côte, pour faciliter leur comparaison. La conception d’Hyperglosae respecte le principe de responsabilité environnementale.
- **Contrairement à :** d’autres outils tels que Microsoft Word
- **Notre produit :** Permet la création libre de liens hypertextes bidirectionnels entre différentes sources par les lecteurs et auteurs de documents culturels

**Comment by vernet-william** [comment]
Voilà ma vision produit du projet Hyperglosae exprimée avec la manière “vision product statement”.
**Pour** les utilisateurs adeptes des manières traditionnelles de rédiger un document avec des références.
**Qui** veulent rédiger des documents ayant des références liées de manière bidirectionnelle.
**Hyperglosae** est un éditeur de texte
**Qui** utilise des liens bidirectionnelles pour rédiger des documents.
**Contrairement** à des éditeurs tels que Google Docs
**Hyperglosae** apporte la possibilité à l’utilisateur de sourcer des oeuvres avec d’autres documents de manière bidirectionnel.

**Comment by Barnabepetit** [comment]
En partant du product vision statement tel que vu jusqu'a maintenant, ma vision du produit Hyperglosae serait :

**Pour** les utilisateurs souhaitant un usage (lecture, édition, liaisons) simplifié de documents.
**Qui** ont besoin d'ajouter et de gérer des référencements des ressources entre elles et les organiser de manière plus logique.
Cet outil offre différentes possibilités, comme des annotations pratiques, visibles et accessibles lors de la manipulation de ressources (documents, images...), ainsi qu'une navigation bidirectionnelle entre elles.
**A la différence** de certains outils permettant la consultation de documents (PDFelement, Adobe acrobate reader) ou encore l'édition (Framapad, Google docs)
**Notre produit** permet de se déplacer et de consulter, d'éditer librement des ressources associés au document de base, notamment à l'aide d'annotation et de liens hypertexte.

**Comment by koeltv** [comment]
Pour moi, la vision du produit "HyperGlosae" est de créer une infrastructure hypertexte inspirée par les concepts de Theodor Nelson. L'objectif est de permettre la création de liens bidirectionnels entre documents de formats variés de manière à ce que les lecteurs puissent naviguer librement entre les documents liés, la création de nouveaux liens sans validation préalable et la lecture des documents côte à côte.
HyperGlosae ambitionne de devenir un outil essentiel pour la lecture, l'écriture et l'analyse collaborative de documents dans les domaines des humanités. Son objectif ultime est de stimuler la création de nouveaux contenus tout en préservant et en mettant en valeur les documents existants, offrant ainsi une expérience plus riche de la connaissance et de la recherche dans ces domaines.

**Comment by GwenRiou** [comment]
Selon moi, la vision produit de l'HyperGlosae suivant la méthode de « Product Vision Statement » est la suivante :
**Pour** toutes les personnes
**Qui** souhaitent classifier les documents en utilisant une méthode traditionnelle qui visualise leurs connexions.
**L’ HyperGlosae** est un logiciel de création et de classification de documents utilisant des hyperlinks
**Qui** améliore la navigation entre les documents grâce à des liens bidirectionnels pour référencée les documents ainsi qu’à la possibilité d’ouvrir les documents cote à cote.
**Contrairement** à Wikipedia,
**Notre produit** permet à tous, de référencer tous types de documents ainsi que rentre visiblement les liens. 

Son Slogan peut être : 
Permettre des référencements traditionnels dans l’environnement numérique

**Comment by PaulGitProject** [comment]
Theodor Nelson a proposé un concept de liens hypertextes dans des documents. Le projet HyperGlosae cherche donc a implémenter son concept. Si l’on se base sur la vision produit (_product vision statement_), voici ce que HyperGlosae est pour moi :

- **Une simplification** de l’utilisation de différents documents (utilisation = lecture)
- **C’est** un logiciel de création et d’édition de documents
- **Qui** rend la navigation entre les documents plus simple et efficace grâce aux liens hypertext
- **A la différence** de certains outils permettant la consultation ou l'éditionde documents (comme Wikipedia ou Word)
- **Notre produit permet** de se déplacer, consulter et éditer des ressources qui sont liées au document de base à l'aide de liens hypertexte (liens bidirectionnels) sans pour autant avoir besoin d'une validation de l'auteur ou autre personne. 
- HyperGloasae est un outils d'édition collaboratif

**Comment by khbgbgk1** [comment]
Voici pour moi la vision produit de Hyperglosae :
**Pour** tout types d'utilisateurs
**Qui** souhaitent approfondir un sujet ou documenter un sujet
**Hyperglosae** est une infrastructure hypertexte
**Qui** a la particularité d'être bilatérale et modifiable par tous ceux ayant un login
**Pas comme** Wikipédia ou l'encyclopédie Larousse
**Notre produit**  permet donc d'accéder aussi bien aux sources d'un article, commentaires, images mais aussi à toutes références qui utilise ce même document comme source. Il permet aussi de l'enrichir et de revenir dessus au fur et à mesure du temps.

**Comment by EnzoSimonoviciUTT** [comment]
Ma vision produit pour le projet Hyperglosae basé sur la méthode de "Product vision statement" est :

**Pour** tout types d'utilisateurs
**Qui** souhaitent se documenter sur un sujet ou y apporter davantage de connaissances
**Hyperglosae** est une plateforme de liens hypertexte
**Qui** facilite la navigation entre les informations à l'aide de liens bidirectionnelle
**Contrairement à** Wikipédia
**Ce produit offre** à ses utilisateurs une vision plus globale d'un sujet, de par la consultation de nombreux documents de tout types (articles, photos, discussions, ...), mais aussi grâce à la possibilité d'éditer ces documents, et à la navigation facilitée par les liens qui relient logiquement les informations

**Comment by AgatheIV** [comment]
Voici ma vision de produit du projet Hyperglosae : 

Pour des individus recherchant des informations 
Qui veulent avoir une navigation en arborescence pour optimiser leur recherche 
[Hyperglosae](https://github.com/Hypertopic/HyperGlosae#readme) permet de créer une structure avec une plue-value pour l'utilisateur 
A la différence de Google Scholar, Hyperglosae permet de comparer les documents liès 
Notre produit permet de mettre en valeur les liens entre les écrits

**Comment by antooinex** [comment]
Selon moi, la vision du produit _Hyperglosae_ est d'offrir aux internautes un outil permettant de lier librement des documents les uns aux autres, de sorte à ce que toutes les connaissances en relation avec un document en cours de consultation soient accessibles en l'espace d'un clic.

Les liens entre documents sont bidirectionnels, ce qui signifie que pour chaque document référencé dans cet outil, l'utilisateur a deux options de navigation. La première est de naviguer vers un document référencé par le document en cours de consultation, et la deuxième est de naviguer vers un autre document qui référence le document en cours de consultation.
Cet aspect bidirectionnel assure un enchaînement continu de possibilités de navigation (en avant, en arrière, vers une autre branche) puisqu'aucun document ne peut être orphelin, c'est-à-dire ne contenir aucun lien vers un autre document (en effet s'il ne contenait aucun lien alors il ne serait lui-même pas accessible).

**Comment by jeromeBRM** [comment]
Voici ma vision produit en utilisant la méthode Product Vision Statement :

Pour les lecteurs
Qui cherchent à se documenter ou contribuer à un document sur internet,
Hyperglosae est une plateforme de gestions de liens hypertextes
Qui permet de lire, annoter, comparer, citer et mettre en liens des documents.

Contrairement aux encyclopédies en ligne actuelles,
Hyperglosae permet de créer des liens hypertextes bidirectionnels, offrant plus de possibilités de navigation entre les documents référencés.

**Comment by Anto792** [comment]
D'après moi, la vision produit permet de donner un sens à un projet à l'instant t comme dans un futur évolutif. Cette ligne directrice permet d'amener du sens lors de la collaboration entre les individus en posant des thématiques centrales, favorisant ainsi le dialogue entre les équipes de développement et partie prenante.

Des méthodes permettent de poser des cadres à la vision produit comme :
- **La méthode de Product vision statement** : Qui impose une réflexion sur le produit à différente échelle (Quel est le nom du produit, quel est la cible, quel avantage, quel différence avec les produits existants, ...);
- **La méthode de Kanban** : Permet de catégoriser le besoin selon le contexte (**idée suggéré** par la partie prenante, **A faire** les idées à mettre en place, **En cours** les fonctionnalités en cours d'implémentation et **Les taches terminées**).

Hyperglose est une plateforme de lien hypertexte qui permet à une équipe d'accéder à une encyclopédie en privée. Ce qui favorise la consultation et de traitement de ces documents.

**Comment by petitfa1** [comment]
Vision produit : 

Pour les personnes qui travaillent avec des documents en ligne et qui veulent améliorer leur recherche d'informations et leur compréhension,

Qui ont besoin d'ajouter des références et des liens pertinents dans leurs documents,

Hyperglosae est une plateforme collaborative qui facilite le référencement des documents numériques,

Contrairement aux outils classiques de traitement de texte en ligne et aux documents qui manquent souvent de liens vers des sources

Notre produit se distingue par sa capacité à permettre aux utilisateurs d'ajouter des liens hypertextes bidirectionnels, favorisant ainsi une meilleure navigation entre les documents et offrant une expérience de lecture plus interactive.

**Comment by louisduhalberruer** [comment]
Ma vision produit du projet Hyperglosae:

Basé sur la vision de Theodor Nelson

Pour les utilisateurs voulant lire des documents plus naturellement, en intégrant les références à proximité lors de la lecture, et, rédiger des documents en intégrant facilement des références.
Qui ont besoin de plus d'informations que le document principal lors de la lecture.
Cet outil offre la possibilité d'une navigation bidirectionnelle et d'informations liées lors de la lecture
À différencier de Wikipédia, Adobe acrobate reader, etc.

Slogan : compléter les connaissances

**Comment by nitram35** [comment]
En utilisant la méthode Product Vision Statement, ma vision produit d'Hyperglosae est la suivante : 

- Pour tout individus 
- Qui souhaitent s'informer/se documenter
- Hyperglosae est une infrastructure hypertexte
- Qui utilise des liens bidirectionnelle facilitant la navigation entre les différentes informations
- Contrairement à certains outils d'édition de documents ou de consultation d'informations (Wikipédia/Word)
- Ce produit permet la consultation, le déplacement, le libre référencement de tous types de documents associés à un document de base et une liberté d'édition de commentaires ou de liens hypertextes.

**Comment by mbel-jordan** [comment]
selon moi la vision du produit sous la forme d'un slogan serait : **Révolutionner le Partage de Connaissances à travers une Infrastructure Hypertexte Avancée**

**Comment by Odyrac** [comment]
Personnellement, je pense qu'une vision produit d'_Hyperglosae_ pourrait être : 

- **Pour** les chercheurs, les étudiants, les écrivains et toute personne passionnée par l'exploration et la connexion des connaissances.
- **Qui** souhaitent naviguer de manière intuitive et efficace à travers un vaste réseau d'informations interconnectées.

_Hyperglosae_ est une plateforme,
- **Qui** permet de découvrir des relations inattendues entre les document et de contribuer à un écosystème de savoir partagé.
- **Contrairement** aux moteurs de recherche traditionnels et aux encyclopédies en ligne statiques, qui offrent une vision linéaire et fragmentée de l'information.

**Comment by claragullotta** [comment]
- Pour tous ceux désireux d'approfondir leur connaissance d'un document
- Qui souhaitent accéder facilement aux références et enrichir la réflexion sur le sujet grâce à leurs contributions.

Hyperglosae est un outil collaboratif qui permet la navigation et l'édition dans un réseau de documents structuré par les liens hypertexte,

- Contrairement aux encyclopédies collaboratives comme Wikipédia,
- Notre produit permet de situer chaque œuvre dans un corpus documentaire, et facilite l'analyse conjointe des documents (grâce à l'affichage simultané du document et d'une de ses gloses par exemple).

**Comment by benel** [comment]
Merci @Odyrac et @claragullotta pour vos contributions à la vision produit.

Par contre, je n'ai pas compris ce qui, dans les documents que je vous ai transmis (manuel d'utilisation et surtout vidéos des projets des années précédentes) vous fait penser qu'*Hyperglosae* serait une alternative aux encyclopédies en ligne... Aucun des domaines d'application des projets des années précédentes ne me semble pouvoir être fait sur Wikipédia, si ? Ou peut-être vouliez-vous dire un "wiki" (de manière générale) ?

**Comment by floporsch911** [comment]
Pour les étudiants, les élèves, les professeurs, les journalistes
Qui souhaite rédiger, éditer, analyser ?

Hyperplasie est une plateforme,

Qui est collaborative et permet de faire des liens entre les documents, d'avoir une connexion logique, claire et précise pour retrouver, suivre un cheminement de pensée.
Contrairement à un éditeur de texte où l'on ne peut que mettre des commentaires uniques. 
Notre produit permet de faire des liens plus profonds, clairs entre les sources.

**Comment by lunatiique** [comment]
**Pour** les étudiants, les enseignants, les amateurs d'arts et les chercheurs
**Qui** souhaitent analyser, comparer, commenter et lier des documents
**Hyperglosae** est une plateforme proposant de naviguer et contribuer à un réseau de documents de manière collaborative.
**Contrairement** aux éditeurs ou lecteurs de documents traditionnels (Word, Acrobat Reader),
**Notre outil permet** de situer et d'éditer un document dans son contexte de manière dynamique, en respectant les différents liens entre eux.

**Comment by Sidqui-Youssef** [comment]
De mon point de vue, la vision produit d'*Hyperglosae* est la suivante : 

- **Pour** les étudiants, les enseignants, les chercheurs, les journalistes 

- **Qui** souhaitent créer, éditer et naviguer à travers des documents liés par des informations interconnectés

*Hyperglosae* est une plateforme collaborative d'édition et de classification de documents grâce à l'utilisation de liens hypertextes. 

- **Qui** permet une meilleure navigation entre les documents grâce aux liens bidirectionnelles afin d'avoir une visualisation claire des relations entre les informations

- **Contrairement** à un éditeur de document classique (Word) ou à des encyclopédies en ligne (Wikipédia)

- **Notre produit** se démarque par sa capacité de pouvoir consulter, de naviguer et de référencer librement tous types de documents associés à un document principal.

- **Slogan** : Liez, naviguez, explorez vos documents autrement.

**Comment by Amine-jabote** [comment]
Selon moi, une vision produit d'Hyperglosae pourrait être la suivante :

**Pour** les lecteurs et les rédacteurs de documents (les étudiants universitaires, enseignants, chercheurs)
**Qui** souhaitent créer, éditer,étudier, interpréter et naviguer à travers des documents liés par des informations structurées
**Hyperglosae** est une plateforme collaborative d'édition et de classification de documents
**Qui** facilite la lecture et l'écriture ainsi que la navigation entre les contenus grâce à des liens bidirectionnels
**Contrairement** aux autres outils d'édition de document classique comme Word ou à des bases de connaissances en ligne, permettant uniquement la consultation et l'édition des documents
**Hyperglosae** se distingue par sa capacité à comparer, consulter, naviguer intuitivement et référencer librement tous types de document hiérarchiquement liés à une source **(document)** principale.

Un **slogan** qui pourrait convenir à notre vision de produit est :

_"Optimisez la gestion et l’exploration de vos documents interconnectés"_

**Comment by florianbnl** [comment]
Vision Produit d'Hyperglosae

"Un écosystème de pensée collaborative où chaque contribution enrichit l'évolution d'un texte."

Énoncé de Vision Produit

Pour toutes les personnes souhaitant participer à une réflexion collective, qu'elles souhaitent créer, modifier ou explorer un texte, Hyperglosae est un éditeur collaboratif qui permet de naviguer de manière bidirectionnelle à travers chaque modification. Contrairement aux outils classiques comme Word, Hyperglosae ne se contente pas d’enregistrer des versions : il révèle le cheminement de la pensée, permettant de suivre et comprendre l’évolution de chaque idée au fil des contributions.

**Comment by cnxsolutions** [comment]
La vision produit est une déclaration stratégique qui définit l'objectif global et la finalité d’un produit. Cela sert de structure pour orienter le développement du produit en alignant différents pôle d'une équipe orientée informatique et marketing.

**Comment by FredWantou** [comment]
La vision produit est l'objectif , la raison pour laquelle on crée un produit permettant d'assurer la continuité d'un produit dans un environnement en constant changement.

En ce qui concerne Hyperglosae, il s'agit d'une plateforme collaborative et de gestion de connaissance
Pour les étudiants, les sociologues, les philosophes
Qui ont besoin d'un outil permettant la gestion et le suivi des connaissances
Hyperglosae est une plateforme qui permet de naviguer à travers le réseau des documents ,créer et éditer un document
contrairement aux outils classiques tels que word permettant juste l'édition de documents
Notre produit permet de créer une chaine de connaissance à travers différents documents.

Le slogan serait Une plateforme au service de la connaissance.

**Comment by Eyvannn** [comment]
Pour les étudiants, enseignants, chercheurs et passionnés de culture.

Qui souhaitent comparer ou analyser des documents et leurs connexions.

Hyperglosae est une plateforme collaborative d'édition et de référencement de documents, permettant ainsi l'interconnexion et l'interprétation collaborative de documents.

Contrairement aux encyclopédies en ligne, qui se contentent de faciliter l'accès aux connaissances et savoirs.

Hyperglosae : Connectez les documents, reliez les esprits.

**Comment by SanKson7** [comment]
**Pour** les étudiants, enseignants et chercheurs.

**Qui** souhaitent une nouvelle façon de créer, étudier et relier des documents.

**HyperGlosae** est une plateforme collaborative.

**Qui** permet de créer et d’éditer des documents, mais aussi de les relier, de les annoter, de les comparer et d’en explorer les connexions.

**Contrairement** aux outils classiques comme Google Docs ou Microsoft Word.

**HyperGlosae** met en avant les liens entre les documents, les annotations et la navigation entre les contenus de manière à garder le schéma de raisonnement suivi depuis la racine.
